analytics/js_tracking_api.rst

```diff
@@ -839,3 +839,19 @@ Example of usage::
 
     _paq.push(["trackSiteSearch", "test", false, 20]);
 
+
+
+Tracking user anonymously
+``````````````````````````
+To track visitor anonymously (without consent) use the ``setUserIsAnonymous`` function::
+
+    _paq.push(["setUserIsAnonymous", isAnonymous]);
+
+.. describe:: isAnonymous
+
+    **Required** ``integer/boolean`` Flag that sets anonymous tracking on and off (true/false or 1/0)
+
+Example of usage::
+
+    _paq.push(["trackSiteSearch", true]);
```

Comment:
Please change trackSiteSearch to setUserIsAnonymous

Reply:
F***...